executor/src/benchmark.rs

```diff
@@ -226,6 +234,17 @@ impl TransactionExecutor {
                 .unwrap();
 
             self.committed_trees = new_committed_trees;
+
+            let commit_time = std::time::Instant::now().duration_since(commit_start);
+            let total_time = execute_time + commit_time;
+
+            info!(
+                "Version: {}. execute time: {} ms. commit time: {} ms. TPS: {}.",
+                version,
+                execute_time.as_millis(),
+                commit_time.as_millis(),
+                num_txns as u128 * 1_000_000 / total_time.as_micros(),
```

Comment:
very unlikely but total_time.as_micros() could underflow to zero and this becomes a divide by zero error.

Reply:
@zamsden micro seconds? I think it's too unlikely to worry about in this case..

Reply:
Would it hurt to add 1 microsecond just to be sure?  A benchmark with a mocked out backend might well complete in a few hundred nanos, in which case the micros will underflow.

Reply:
I changed it to nanos anyway.